nucypher/blockchain/eth/interfaces.py

```diff
@@ -386,7 +386,7 @@ def send_transaction(self,
             # TODO: #1504 - Handle validation failures for gas limits, invalid fields, etc.
             # Note: Geth raises ValueError in the same condition that pyevm raises ValidationError here.
             # Treat this condition as "Transaction Failed".
-            error = str(e).replace("{", "{{").replace("}", "}}")  # See #724
+            error = str(e).replace("{", "").replace("}", "")  # See #724
```

Comment:
:question:

Reply:
Yeah, some example of what's in `e` wouldn't hurt in the comments